app/components/shared/Panel/index.js

```diff
@@ -39,9 +40,9 @@ Panel.IntroWithButton = IntroWithButton;
 Panel.Row = Row;
 Panel.RowActions = RowActions;
 Panel.RowLink = RowLink;
+Panel.Header = Header;
 
 const SIMPLE_COMPONENTS = {
-  Header: 'bg-silver py2 px3 semi-bold',
```

Comment:
Just to clarifiy, I'm talking about how this code assumes a component only has two things: an identifier, and a list of class names.

Whereas if we just always used `React.PureComponent` for things, and made them declarative in JSX, then everything in the system is the same, and it's easy for anyone to, say, change a tag name, or a class name, or add an event handler, etc. And if we want to DRY some logic into a single place, we do that using our own functions calls from the components?

I just always find I regret defining classes/functions/things imperatively in loops like this, as it just makes it hard for anyone following me to fix/customize/adapt things.